SPECS/python-virtualenv/python-virtualenv.spec

```diff
@@ -38,7 +43,77 @@ Provides:       %{name}-doc = %{version}-%{release}
 virtualenv is a tool to create isolated Python environment.
 
 %prep
-%autosetup -p1 -n virtualenv-%{version}
+# Adding -N to enable manual patching, needed for CVE-2025-50181 
```

Comment:
I think we should probably find a better way of patching this CVE.
Dropping `autosetup` might not be a good.
Can we please do some research on what are the files that are actually shipped so that we can patch only those.